wondrous-app/components/Header/index.tsx

```diff
@@ -26,6 +30,21 @@ const HeaderComponent = () => {
   ];
   const showCreateButton = urlsWithCreateButton.some((url) => router.pathname?.includes(url));
 
+  useEffect(() => {
+    if (user && !user?.userInfo?.discordUsername && router.pathname !== '/profile/settings') {
+      setSnackbarAlertOpen(true);
+      setSnackbarAlertAutoHideDuration(DISCORD_SNACKBAR_DURATION);
+      setSnackbarAlertMessage(
+        <>
+          <ConnectDiscordLink href="/profile/settings" target="_blank" rel="noreferrer">
+            Connect your Discord
+          </ConnectDiscordLink>{' '}
+          <span>for notifications for upcoming tasks, comments and more!</span>
```

Comment:
since this shows up show much maybe shorten it,
for notifications on tasks, comments and more!